src/app/containers/PageHandlers/withOptimizelyProvider/index.jsx

```diff
@@ -5,14 +5,12 @@ import {
   setLogger,
 } from '@optimizely/react-sdk';
 import { ServiceContext } from '#contexts/ServiceContext';
-import isLive from '#lib/utilities/isLive';
 import { GEL_GROUP_3_SCREEN_WIDTH_MAX } from '@bbc/gel-foundations/dist/breakpoints';
 import onClient from '#lib/utilities/onClient';
 import getOptimizelyUserId from './getOptimizelyUserId';
 
-if (isLive()) {
-  setLogger(null);
-}
+// 004_brasil_recommendations_experiment
+setLogger(null);
```

Comment:
The logger wasn't disabled on local/test env as it does provide some useful optimizely output (such as the variant a user is in), it is disabled on LIVE though. The cypress tests are failing because on the initial render, optimizely object is null and an error is thrown which cypress detects and throws an error. 

More context [here](https://bbcnews.slack.com/archives/C02UABCDD2A/p1654076713590999) and [here](https://bbcnews.slack.com/archives/C02UABCDD2A/p1653990779516369).

Solution is to temporarily disable the optimizely logging.